tools/bench/_timing.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import csv
+import time
+
+
+def timeit(lst):
```

Comment:
added dataclasses as you suggested... it is a bit better, but still not awesome.  as things are added here, this can be improved.   It isnt clear how much it is going to evolve right now.